index.bs

```diff
@@ -1040,6 +1185,111 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
       {{Screen}}.
     </p>
   </section>
+
+  <section>
+    <h3 id='video-display-capabilities'>Video Display Capabilities</h3>
+
+    <section>
+      <h4 id='video-display-configuration'>VideoDisplayConfiguration</h4>
```

Comment:
I think we actually don't need a config/info style here anymore - you could just make width/height/hdrSupported attributes of window.screen.video. IIRC the method comes from earlier thinking when we thought we were going to be querying for more granular features like colorGamut, for which an attribute doesn't work as well. With those gone it seems silly to have a method.

Reply:
fyi ^^ @jernoble @mounirlamouri @jyavenard

Reply:
and fyi @mwatson2 (you're already reading along here, but just in case this gets buried in my comments)

Reply:
@chcunningham It's best to keep this as a query due to the fingerprinting analysis that we did earlier which will allow mitigations if necessary. @jernoble @mwatson2

Reply:
@gregwhitworth - can you give more details? what mitigations can we provide via method that would not be possible for the attributes?

Taking a fresh look at the fingerprinting sections now...

> Because these features require directly calling asynchronous Web APIs, they are very detectable by the UA, and does not represent risk of passive fingerprinting.

UA is no less aware of calls on attributes. Maybe the thinking was that attributes in JS are just simple read-only attributes all the way down. But, at least for WebKit based UAs, this is never the case; C++ doesn't have this style of attribute. For example: take `<video>.currentTime`,  [defined as an attribute](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/media/html_media_element.idl?rcl=c281c2a54fee5f392ffb5359ffef5525881be4a9&l=63), but [implemented as a method](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/media/html_media_element.cc?rcl=c281c2a54fee5f392ffb5359ffef5525881be4a9&l=2220). 

> If an implementation wishes to implement a fingerprint-proof version of this specification, it would be recommended to fake a given set of capabilities (ie. decode up to 1080p VP9, etc.) instead of returning always yes or always no as the latter approach could considerably degrade the user’s experience. Another mitigation could be to limit these Web APIs to top-level browsing contexts. Yet another is to use a privacy budget that throttles and/or blocks calls to the API above a threshold.

I don't think any of these mitigations require a method?

Reply:
Can we move this discussion here for resolution: https://github.com/w3c/media-capabilities/issues/134